standard/template/src/STRUCTURE.md

```diff
@@ -1 +1,12 @@
-# TODO: add this :)
+# Folder Structure
+
+The structure follows [atomic design](https://bradfrost.com/blog/post/atomic-web-design/)
+guidelines.
+
+- atoms - the smallest unit, must be self contained and not dependant on any external modules.
+  (types and configuration being the exception)
+  - `Button`, `Paragraph`, `Heading`, etc..
```

Comment:
Button might not be an atom if it consist out of label and icon atoms :)
So another "rule" should be that it doesn't import any other components itself.

Reply:
Fair point, a `Button` can even be an organism if it gets crazy enough.
That rule is already present:
> not dependant on any external modules. (types and configuration being the exception)

modules == components